api/MediaSource.json

```diff
@@ -0,0 +1,974 @@
+            "firefox": [
+              {
+                "version_added": "42"
+              },
+              {
+                "version_added": "25",
+                "version_removed": "42",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.mediasource.enabled"
+                  }
+                ],
+                "notes": [
+                  "Limited support to a whitelist of sites, for example YouTube, Netflix, and other popular streaming sites. The whitelist was removed when Media Source Extensions was enabled by default in Firefox 42."
+                ]
+              }
+            ],
+            "ie": {
+              "version_added": "11",
+              "notes": [
+                "Only works on Windows 8+."
+              ]
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "8"
+            },
+            "webview_android": {
+              "version_added": "33"
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "41"
+            },
+            "opera_android": {
+              "version_added": "30"
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "setLiveSeekableRange": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/setLiveSeekableRange",
+          "support": {
+            "chrome": {
+              "version_added": "62"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "clearLiveSeekableRange": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/clearLiveSeekableRange",
+          "support": {
+            "chrome": {
+              "version_added": "62"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```

Comment:
This should be 49. Please also add `opera_android` with 49 as the version number.